libs/service-portal/graphql/src/lib/queries/getUsersVehicleDetail.ts

```diff
@@ -0,0 +1,98 @@
+import { gql } from '@apollo/client'
+
+export const GET_USERS_VEHICLE_DETAIL = gql`
+  query GetUsersVehiclesDetail($input: GetVehicleDetailInput!) {
```

Comment:
I know this is done almost everywhere and should be tackled as a whole.  But this is taking the power of GraphQL away and treating it like a REST API. Here, everything is queried, regardless of it's needed or not, instead of allowing the components to query what is needed in individual components, and thus allowing e.g. batching types into single requests, and skipping heavy queries when not needed, etc.

Reply:
Thanks for the info, didnt realise it. I moved it to a seperate folder inside /libs/service-portal/vehicles instead.